TestFoundation/TestCodable.swift

```diff
@@ -33,35 +33,42 @@ private func makePersonNameComponents(namePrefix: String? = nil,
     return result
 }
 
-func expectRoundTripEquality<T : Codable>(of value: T, encode: (T) throws -> Data, decode: (Data) throws -> T) throws where T : Equatable  {
+func expectRoundTripEquality<T : Codable>(of value: T,
+                                          encode: (T) throws -> Data,
+                                          decode: (Data) throws -> T) throws where T : Equatable  {
```

Comment:
Let's not reformat all of these functions.

Reply:
Do we have any guides for line lengths?

Reply:
In Foundation code, we avoid explicit wraps except under very specific circumstances (preferring to leave it up to the editor to wrap as appropriate).

Reply:
Oh, good to know. I'll stick to this rule, thanks.